plugins/tpm/fu-plugin-tpm.c

```diff
@@ -29,9 +29,12 @@ void
 fu_plugin_device_added (FuPlugin *plugin, FuDevice *dev)
 {
 	FuPluginData *data = fu_plugin_get_data (plugin);
+	const gchar *version = fu_tpm_device_get_family (FU_TPM_DEVICE (dev));
+
 	data->has_tpm = TRUE;
-	if (g_strcmp0 (fu_tpm_device_get_family (FU_TPM_DEVICE (dev)), "2.0") == 0)
+	if (g_strcmp0 (version, "2.0") == 0)
 		data->has_tpm_v20 = TRUE;
+	fu_plugin_add_report_metadata (plugin, "TpmVersion", version);
```

Comment:
This is really TPM family not TPM version no?

Reply:
I have no idea why they call it a family. It's a string like "1.2" or "2.0"

Reply:
Maybe the report metadata should also be `TpmFamily` to avoid confusion?

Reply:
>I have no idea why they call it a family. It's a string like "1.2" or "2.0"

Well it's indicating how it behaves, not a firmware version.

>Maybe the report metadata should also be TpmFamily to avoid confusion?

Yes

Reply:
Fair enough, agreed. I've pushed something that might pass muster.